### PR TITLE
Add mentor bio for ruby track

### PR DESCRIPTION
**Pull request by dipto0321**

### CODE REVIEW

File: mentors/ruby/d.json [hunk 1]
```diff
@@ -14,5 +13,13 @@
...
+  {
+    "github_username": "dipto0321",
+    "name": "Dipto Karmakar",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I am passionate about programing. Love to build new things with Ruby, python and Javascript. I am enjoying too much by sharing my knowledge and learning from others. Love to contribute on open source projects"
   }
-]
+]
```
Comment (rpottsoh) [inline mentors/ruby/d.json]: Try to avoid making changes that are outside of the scope of the PR.

File: mentors/ruby/d.json [hunk 2]
```diff
@@ -1,5 +1,4 @@
-[
-  {
+[{
```
Comment (rpottsoh) [inline mentors/ruby/d.json]: Try to avoid making changes that are outside of the scope of the PR.  The two unnecessary changes I have flagged are minor so I will go ahead and provide approval for this PR.

Comment (dipto0321) [reply mentors/ruby/d.json]: Thanks. In the future, I will be aware of these.
